i2cDevice: /dev/i2c-1
frequency: 50.0
minFreq: 30.0
maxFreq: 60.0
